srcpkgs/dosbox-x/template

```diff
@@ -0,0 +1,41 @@
+# Template file for 'dosbox-x'
+pkgname=dosbox-x
+version=2025.02.01
+revision=1
+build_style=gnu-makefile
+build_helper=qemu
+hostmakedepends="pkg-config which autoconf automake nasm"
+makedepends="SDL2-devel SDL2_net-devel
+ libpcap-devel libslirp-devel fluidsynth-devel libavdevice6
+ libavformat6 libavcodec6 libswscale6
```

Comment:
the non `-devel` libraries seem wrong here.

Reply:
Actually, there are no `-devel` versions for these libraries @Duncaen

Reply:
You probably want `ffmpeg6-devel` in this case if it links against any of those libraries.

```
ffmpeg6-devel-6.1.2_6        /usr/lib/libavcodec.so
```